modules/phase_field/include/vectorpostprocessors/GrainCentersPostprocessor.h

```diff
@@ -0,0 +1,80 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#ifndef GRAINCENTERSPOSTPROCESSOR_H
+#define GRAINCENTERSPOSTPROCESSOR_H
+
+#include "GeneralVectorPostprocessor.h"
+#include "ComputeGrainCenterUserObject.h"
+
+//Forward Declarations
+class GrainCentersPostprocessor;
+
+template<>
+InputParameters validParams<GrainCentersPostprocessor>();
+
+/**
+ *  GrainCentersPostprocessor is a type of VectorPostprocessor that outputs center and volume of grains
+ *  calculated in GrainCenterUserObject.
+ */
+
+class GrainCentersPostprocessor :
+  public GeneralVectorPostprocessor
+{
+public:
+  /**
+    * Class constructor
+    * @param name The name of the object
+    * @param parameters The input parameters
+    */
+  GrainCentersPostprocessor(const InputParameters & parameters);
+
+  /**
+   * Destructor
```

Comment:
Let's remove all "boilerplate" comments and retain only the ones that are
specific to your classes.

On Mon, Aug 3, 2015 at 9:45 AM, Sudipta Biswas <notifications@github.com>
wrote:

> In
> modules/phase_field/include/vectorpostprocessors/GrainCentersPostprocessor.h
> <https://github.com/idaholab/moose/pull/5499#discussion_r36098393>:
>
> > + *  calculated in GrainCenterUserObject.
> > + */
> > +
> > +class GrainCentersPostprocessor :
> > +  public GeneralVectorPostprocessor
> > +{
> > +public:
> > +  /**
> > +    * Class constructor
> > +    * @param name The name of the object
> > +    * @param parameters The input parameters
> > +    */
> > +  GrainCentersPostprocessor(const InputParameters & parameters);
> > +
> > +  /**
> > +   * Destructor
>
> Do you want only the '// Destructor' comment to be removed or all of these
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5499/files#r36098393>.
>